_people/lislis.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Lisa Passing"
+groups:
+  - committee
+links:
+  - icon: github
+    link: https://github.com/lislis
+priority: 60
+---
+
+Lisa is a software developer and hobbyist game creator and creative
```

Comment:
Seems weird that the line is cut here whereas it's not in other descriptions, no?

Reply:
Good catch, removed the newline.